cdap-ui/app/cdap/styles/variables.scss

```diff
@@ -153,6 +153,8 @@ $green-04: #0f9d58;
 $green-05: #0b8043;
 
 :export {
+  white01: $cdap-white;
```

Comment:
should we add this both to the less and sass?

Reply:
Don't know if its needed. Right now we export css variables and use it in js only in react side. Do you see anywhere in angular side where we need to use?